app/classes/inat/obs.rb

```diff
@@ -219,15 +221,13 @@ def snapshop_raw_str
         ID: inat_taxon_name,
         DQA: dqa,
         show_observation_inat_suggested_ids: suggested_id_names,
-        OBSERVATION_FIELDS: obs_fields(inat_obs_fields),
-        PROJECTS: :inat_not_imported.t,
-        ANNOTATIONS: :inat_not_imported.t,
-        TAGS: :inat_not_imported.t
+        OBSERVATION_FIELDS: obs_fields(inat_obs_fields)
       }.each do |label, value|
-        result += "#{label.to_sym.t}: #{value}\n"
+        result += "#{label.to_sym.t} #{value}\n"
       end
-      result
+      result.chomp
     end
+    private :snapshop_raw_str
```

Comment:
Is this a way to declare a single method private? Neat if so, i didn't know.

Reply:
Thanks for the review. 
Yes, that's a way to declare a single method (or an array of methods) `private`.
https://docs.ruby-lang.org/en/3.3/Module.html#method-i-private

Reply:
@JoeCohen 

- The meaning of "Snapshot" is not obvious to me. What if you call it something that makes it clear where it came from, like "iNaturalist imported data:"?
- I can see why the formatting of this has been difficult. I think i would prefer putting a newline at the beginning, so "User" starts on a new line, and using colons after every sub-field, but just not underlining the sub-fields. That way it's clear where each field is, but it doesn't interfere with scanning the major Obs notes fields. 

So like this
___
Notes:
  <ins>Collector</ins>: jdcohenesq
  <ins>iNaturalist imported data</ins>:
  User: jdcohenesq
  Observed: 2024-09-04
  Public lat/lng: 234.25325434235
  Place: Skamania County, US-WA, US
  ID: Mycena
  Quality Grade: Needs ID
  etc